pkg/client/chains.go

```diff
@@ -0,0 +1,133 @@
+package client
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	testnetURL = "https://testnet.tableland.network"
+	localURL   = "http://localhost:8080"
+)
+
+// ChainID is a supported EVM chain identifier.
+type ChainID int64
+
+// ChainIDs is all chain ids supported by Tableland.
+var ChainIDs = struct {
+	Ethereum       ChainID
+	Optimism       ChainID
+	Polygon        ChainID
+	EthereumGoerli ChainID
+	OptimismKovan  ChainID
+	OptimismGoerli ChainID
+	ArbitrumGoerli ChainID
+	PolygonMumbai  ChainID
+	Local          ChainID
+}{
+	Ethereum:       1,
+	Optimism:       10,
+	Polygon:        137,
+	EthereumGoerli: 5,
+	OptimismKovan:  69,
+	OptimismGoerli: 420,
+	ArbitrumGoerli: 421613,
+	PolygonMumbai:  80001,
+	Local:          31337,
+}
```

Comment:
Got tired of polluting the top level namespace of the file with many `Chain` vars, so trying this approach to wrap them in a `struct`. Let me know if this is somehow offensive, but I like it I think.